src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.domain
+
+class Lotto private constructor(private var numbers: List<Int>) {
```

Comment:
기본적으로 `val` 키워드를 사용하여 모든 변수를 읽기 전용으로 선언하고 필요할 때만 `var`로 변경하면 어떨까요?

```suggestion
class Lotto private constructor(private val numbers: List<Int>) {
```